src/main/resources/assets/app/scripts/templates/job_detail_view.hbs

```diff
@@ -55,20 +70,40 @@
 
     <div class="row-fluid job-detail-row">
       <div class="span6">
-        <div class="control-group job-detail"
-             data-toggle="tooltip"
-             data-title="Owner's email address(es)<br>The owner(s) will be emailed in the event of a job failure or disruption."
-             data-html="true"
-             data-placement="bottom">
-          <div class="detail">Owner(s)</div>
-          <div class="display hightlight owner"
-               data-rv-text="job.owner">{{owner}}</div>
-          <div class="field cmd">
-            <input class="job-input" value="{{owner}}" name="owner" tabindex="4"
-                   data-rv-value="job.owner">
+        <div class="row-fluid">
+          <div class="control-group job-detail"
+               data-toggle="tooltip"
+               data-title="Owner's email address(es)<br>The owner(s) will be emailed in the event of a job failure or disruption."
+               data-html="true"
+               data-placement="bottom">
+            <div class="detail">Owner(s)</div>
+            <div class="display hightlight owner"
+                 data-rv-text="job.owner">{{owner}}</div>
+            <div class="field cmd">
+              <input class="job-input" value="{{owner}}" name="owner" tabindex="6"
```

Comment:
looks like the tab index should be 5, please check the rest to see if they line up as well.